patternfly/patternfly-elements
# docs: updating dev server header to match live site

## What I did

1. Updated dev server header to match 11ty doc site styles
2. Added `repoHost` to `PfeConfig` which enables customizing the dev server repository host name and icon

## Testing Instructions

1. Checkout branch, run `npm run start`
2. Add a `.pfe.config.json` with the following JSON, and the Icon should change to GitLab.  Adding a blank string for `repoHost` should result in a generic Git logo.
```json
{
  "site": {
    "repoHost": "GitLab"
  }
}
```

## Notes for Reviewers

1. Ensure that the new header works across varying screen sizes.
2. Suggestions welcome on if the configuration of `repoHost` should exist in the `site` block or in at the same level as `sourceControlURLPrefix` and `demoURLPrefix` in `.pfe.config.json` see [RHDS's config](https://github.com/RedHat-UX/red-hat-design-system/blob/main/.pfe.config.json) as a reference of a filled out JSON config file.
